tests/LanguageServerTest.php

```diff
@@ -36,7 +36,10 @@ public function testInitialize()
                 'textDocumentSync' => TextDocumentSyncKind::FULL,
                 'documentSymbolProvider' => true,
                 'hoverProvider' => true,
-                'completionProvider' => null,
+                'completionProvider' => (object)[
+                    'resolveProvider' => false,
+                    'triggerCharacters' => ['$', '>']
```

Comment:
Obviously it can be read in different ways. So a clarification in the protocol would be helpful.